dataland-frontend/tests/component/components/resources/frameworkDataSearch/p2p/P2pPanel.cy.ts

```diff
@@ -0,0 +1,202 @@
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
+      created() {
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-ignore
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-call,
+        this.convertP2pDataToFrontendFormat();
+      },
+    });
+    cy.get(`span[data-test=General]`).click();
+    cy.get(`span[data-test=_general]`).click();
+    cy.get("td:contains('Ammonia')").should("exist");
+  });
+
+  /**
+   * Toggles the data-table row group with the given key
+   * @param groupKey the key of the row group to expand
+   */
+  function toggleRowGroup(groupKey: string): void {
+    cy.get(`span[id=${groupKey}]`).siblings("button").last().click();
+  }
+
+  it("Check P2p view page for company with one P2p data set", () => {
+    const preparedFixture = getPreparedFixture("one-p2p-data-set-with-three-sectors", preparedFixtures);
+    const p2pData = preparedFixture.t;
+
+    cy.intercept("/api/data/p2p/mock-data-id", {
+      companyId: "mock-company-id",
+      reportingPeriod: preparedFixture.reportingPeriod,
+      data: p2pData,
+    } as CompanyAssociatedDataPathwaysToParisData);
+    cy.mountWithPlugins(P2pPanel, {
+      keycloak: minimalKeycloakMock({}),
+      global: {
+        stubs: {
+          transition: false,
+        },
+      },
+      data() {
+        return {
+          companyId: "mock-company-id",
+          singleDataMetaInfoToDisplay: {
+            dataId: "mock-data-id",
+            reportingPeriod: preparedFixture.reportingPeriod,
+          } as DataMetaInformation,
+        };
+      },
+    });
+
+    cy.get(`span.p-column-title`).should("contain.text", p2pData.general.general.dataDate.substring(0, 4));
+    cy.get(`span[data-test=General]`).click();
+    toggleRowGroup("_general");
+    cy.get("tbody").find(`span:contains(${p2pData.general.general.dataDate})`).should("exist");
+
+    toggleRowGroup("_general");
+    cy.get("tbody").find(`span:contains(${p2pData.general.general.dataDate})`).should("not.exist");
+
+    toggleRowGroup("_general");
+    cy.get("table.p-datatable-table").find(`span:contains(${p2pData.general.general.dataDate})`).should("exist");
+
+    cy.get(`span[data-test=Ammonia]`).click();
+
+    cy.get("span[data-test=ccsTechnologyAdoption]").should("not.exist");
+    toggleRowGroup("decarbonisation");
+    cy.get("span[data-test=ccsTechnologyAdoption]").should("exist");
+
+    cy.get(`span[data-test="Livestock farming"]`).click();
+    toggleRowGroup("animalFeed");
+    cy.get("span[data-test=Report-Download-Policy]").find("i[data-test=download-icon]").should("be.visible");
+
+    cy.get(`span[data-test=Cement]`).click();
+    toggleRowGroup("material");
+    cy.get("span[data-test=preCalcinedClayUsage]").should("exist");
+
+    cy.get("em[title='Pre-calcined clay usage']").trigger("mouseenter", "center");
+    cy.get(".p-tooltip").should("be.visible").contains("Share of pre-calcined");
```

Comment:
`.contains(...)` is actually different thing from `.should("contain.text", ...)`. The version implemented does not use the element already found but returns an element which contains "Share of pre-calcined", not necessarily a ".p-tooltip". We should adapt this whereever we find such a thing in general.

Reply:
Changed